library/src/main/java/com/novoda/downloadmanager/lib/DownloadDeleter.java

```diff
@@ -25,16 +25,16 @@ public void deleteFileAndDatabaseRow(DownloadInfo info) {
     }
 
     public void deleteFileAndMediaReference(DownloadInfo info) {
-        if (!TextUtils.isEmpty(info.mMediaProviderUri)) {
-            resolver.delete(Uri.parse(info.mMediaProviderUri), null, null);
+        if (!TextUtils.isEmpty(info.getMediaProviderUri())) {
+            resolver.delete(Uri.parse(info.getMediaProviderUri()), null, null);
         }
 
-        if (!TextUtils.isEmpty(info.mFileName)) {
-            deleteFileIfExists(info.mFileName);
+        if (!TextUtils.isEmpty(info.getFileName())) {
+            deleteFileIfExists(info.getFileName());
             ContentValues blankData = new ContentValues();
             blankData.put(Downloads.Impl._DATA, (String) null);
             resolver.update(info.getAllDownloadsUri(), blankData, null, null);
-            info.mFileName = null;
+            info.setFileName(null);
```

Comment:
:+1: I'd rather keep that for another PR, there's a lot in this one and I wanna keep the field encapsulation as simple as possible